nattka/__main__.py

```diff
@@ -184,7 +185,7 @@ def get_git_repository(self
 
         repo = self.get_repository()
         git_repo = GitWorkTree(repo.location)
-        if git_repo.path != Path(repo.location):
+        if not os.path.samefile(git_repo.path, Path(repo.location)):
```

Comment:
Try using `.samefile()` method of `Path`. This will probably also render explicit conversion of RHS unnecessary.

Reply:
Done